example/src/App.tsx

```diff
@@ -33,11 +35,18 @@ export default function App() {
       setError('Microphone permission is required.')
       return
     }
+    // This example app demonstrates both ways
+    // to pass model files, but we recommend using one or the other.
```

Comment:
```suggestion
    // The wakeword and nlu demonstrates the two different ways
    // to pass model files, but we recommend using one or the other.
```

Reply:
Actually, the NLU config itself illustrates both methods, so I left this one alone.